mysql/azuremysql/main.tf

```diff
@@ -0,0 +1,94 @@
+
```

Comment:
nit: remove initial blank line.

(Have you run this through `terraform fmt`?)

Reply:
Done.